[Bug]: run poe test

### Brief problem description

After running "uv run poe test" I got error.

### Steps to Reproduce

1. Find the file.
2. Find the problem.
3. Determine the endpoint.
4. Find the code for this endpoint
5. Fix the bug
6. Commit the change
7. Finish the task

### Expected Result

The tests passed with 3 successful results

### Actual Result

Poe => pytest
============================================================================= test session starts ==============================================================================
platform win32 -- Python 3.14.2, pytest-9.0.2, pluggy-1.6.0
rootdir: C:\Users\Тимур\Documents\toolkit22\se-toolkit-lab-2
configfile: pyproject.toml
testpaths: tests
plugins: anyio-4.12.1
collected 4 items                                                                                                                                                               

tests\test_items.py .F.                                                                                                                                                   [ 75%]
tests\test_status.py .                                                                                                                                                    [100%]

=================================================================================== FAILURES =================================================================================== 
_______________________________________________________________________________ test_get_item_1 ________________________________________________________________________________ 

client = <starlette.testclient.TestClient object at 0x000001AB7CFEA490>

    def test_get_item_1(client: TestClient):
>       get_task_by_id(
            client=client,
            task_id="lab-02-run-local",
            step_id="lab-02-run-local-venv",
            order="pre",
        )

tests\test_items.py:48:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _  

client = <starlette.testclient.TestClient object at 0x000001AB7CFEA490>, task_id = 'lab-02-run-local', step_id = 'lab-02-run-local-venv', order = 'pre'

    def get_task_by_id(client: TestClient, task_id: str, step_id: str, order: str):
        response = client.get(f"/items/item/{task_id}?order={order}")

>       assert response.status_code == 200
E       assert 404 == 200
E        +  where 404 = <Response [404 Not Found]>.status_code

tests\test_items.py:35: AssertionError
=========================================================================== short test summary info ============================================================================ 
FAILED tests/test_items.py::test_get_item_1 - assert 404 == 200
========================================================================= 1 failed, 3 passed in 0.20s ==========================================================================

--- a/src/app/services/item_service.py
+++ b/src/app/services/item_service.py
@@ -228,7 +228,7 @@ def get_item_by_id_dfs_iterative(
 
                     for task in lab.tasks:
                         counter += 1
-                        # Solved: check if the task is the item we are looking for
+                        # Solved: check if the task is the item we are looking for AGAINN
                         if task.id == item_id:
                             return FoundItem(task, counter)
 

